.travis.yml

```diff
@@ -20,14 +20,19 @@ compiler:
   - clang
 
 env:
-  - BUILD=Kodi TOOLS=Autotools
-  - BUILD=Kodi TOOLS=CMake
-  - ADDONS=adsp
-  - ADDONS=audiodecoder
-  - ADDONS=audioencoder
-  - ADDONS=pvr
-  - ADDONS=screensaver
-  - ADDONS=visualization
+  global:
+    - secure: "iVfJQ2Jo7ThtHHI37NtwO1Y0rpt+ITf12BD2SfSrCLNsFdQS5TfXeEAUn7ApToye9xFXr8UhgFtTJqx5+hS7sylK5o7S4Pc4zEsqSQpTgE1e6jY5FM0dqjWzSSi79xAxol9T6Gsfqr9Pyc/JXr2SvzvPvw12FqdUlbhwPEFPfGgfckoLPt+mLrvJajxnmcuo7eIs+n/ZOEOqxri6Ok+7gvmR11e29xMg8ASX9fvJP7MDb242KkT7VtKzy4j5cvFv9/B7ihde6UM8P5ODc01urG0t9FqrZlVB2Sqh9gV15oYAQyHwAXAdItibiga4Cn858Z0vNy/zmAmRSzGhQscCAlDZhiKDtZJ1KAvCvFJKi6CqvZskbhD6Im9DO6ICtp1ochby+28T0MTFCfzPOe5+BbPe8NgyYRL0jaVhzSYae0LihW7L/Mo99soUjRtkM7uJrkcGqRszX3V0j+qO8K9NjzYxfOCbpVBpnNWRY7qx+HBKajm2SAdBGZdawZ43T1DfGPibWrFjgSDuiSCvstjm4p+9pNIi19vfci0XUIBKO95r5GhRczJktJwd7a1Ol1ODWFTCE/xJmFF3lGMh/MrbDiz2PXZnhjFlPYCPTLLTmS0uoYi7ENG+y5QA5mDSOImUDyvj4yc413jyXT3TuhE/VCWrAZvVhaG1d5EZoCfkpJM="
+    - secure: "ed4giID95UTDRX5Q3rFBiGCXbYDdx8nXmZgVkr8BXeQCrIe6SQHzquHDfHDJKOkpKEo7Y9/npKGN5XADuev/0mpxo79wBAAzj4zSTEzWJ9IlypBMpq2ldR/+4j5Z7DG9FoH0lB7xMCI47PWKFOJRFSixZgotXoYJTYaLM+iVuWJNo7K0XlndnG40VMTVyN4sE+swyvIvTBR+L2La08CB5MT+LmLleI58OtwMDLwA6UNRjsZA8Cji85A8YR7w6a46/xVpUhXbIiWuVxEdVLQCRaMgdTSGkEwTooVWlv8tPcyLGJWPqpL7sqCKLfGnupTENJ+NEHEJ+jmefIDAHwhS4zVQ4DGKZrIs3J6IxvVURAs1ONGzInejZ4+I077A09+wbDzHKc2mgHu4rbaKSlfzeoWrUoSV8Ko2VU75GnbT8ZZOJptAeuA7qQLCciIcYhXzQzzlalp3DCJ3QQAoksb0rngC4V5RMXV1M+e0y4FFQLD5RwUXlZFwMEPkCErWfIh0hkiKKw9X/uCq6MUXUdVYtjvZibJwiTF5y6ZNvyGqq95cnFN9oWzUuncvbVALqV5rcQeTs8XOCvb2FZhv4l3r8mTZxFnK4FaMCV+uxqu4VzPd5qXYL2piUUJxUFqTsNTHUP0zchfPHdin8siEDD8q8OTWJVS7imjN3tBK/Moh4sI="
```

Comment:
Place to change for Kodi itself, there is currently access data for my website
need a:
- `travis encrypt FTP_USER=?????`
- `travis encrypt FTP_PASS=?????`

Currently only for my place ftp possible.